GUIDE.md

```diff
@@ -178,11 +178,9 @@ search. By default, when you search a directory, ripgrep will ignore all of
 the following:
 
 1. Files and directories that match glob patterns in these three categories:
-  1. gitignore globs (including global and repo-specific globs).
-  2. `.ignore` globs, which take precedence over all gitignore globs when
-     there's a conflict.
-  3. `.rgignore` globs, which take precedence over all `.ignore` globs when
-     there's a conflict.
+      1. gitignore globs (including global and repo-specific globs).
+      2. `.ignore` globs, which take precedence over all gitignore globs when there's a conflict.
```

Comment:
Could you make sure all lines are wrapped to 79 columns inclusive? Thanks!

Reply:
+